Firestore/Source/Model/FSTPath.h

```diff
@@ -113,6 +116,17 @@ NS_ASSUME_NONNULL_BEGIN
 /** Returns YES if this is the `FSTFieldPath.keyFieldPath` field path. */
 - (BOOL)isKeyFieldPath;
 
+/** Creates and returns a new path from C++ FieldPath.
+ *
+ * @param fieldPath A C++ FieldPath.
+ */
++ (instancetype)fromCPPFieldPath:(const firebase::firestore::model::FieldPath &)fieldPath;
```

Comment:
nit: for future reference, `fromCppFieldPath` would probably better. But since these are temporary functions that will be removed shortly, I don't care. No action required.

"When abbreviations appear in such names, prefer to capitalize the abbreviations as single words (i.e. StartRpc(), not StartRPC())."

http://go/cpp-style#General_Naming_Rules

Reply:
ack. Here actually use objc naming since it is objc method.

Reply:
oh, good point. Carry on then. :)

Reply:
The initialism is spelled correctly for Objective-C, but the method name doesn't conform otherwise since it's supposed to start with the kind of thing it returns. In this case this should be

```objc
+ (instancetype)fieldPathWithCPPFieldPath(const firebase::firestore::model::FieldPath &)fieldPath;
```

This extreme verbosity is required in Objective-C because there's no name mangling (it's all C linkage). For future interop, I'd suggest making an inline `ToObjc()` method guarded by `#if __OBJC__` on the C++ object instead.

Reply:
ack. BTW: I slightly prefer to keep the temporary conversion methods inside objc file. So it will be removed together with the FST* class once the port is done and objc files removed.

Reply:
SG. Not a big deal here, but I fear that in other cases these conversions may be relatively long-lived. When that's the case I think the extreme verbosity of the objc version harms readability enough to warrant opting for a more compact form, if possible.

For example: @gsoltis is rewriting `FSTWriteGroup` but we're not going to be able to break the dependency on `GPBMessage` until we're ready to cut over to the alternative nanopb implementation. The bridge will exist for a nontrivial amount of time.

Meanwhile, we're already going to have to audit all of the things this finds anyway:
```
find Firestore/core -type f | xargs grep __OBJC__
```
Shows a number of utilities and bridges we've already written for ourselves. I wouldn't worry too much that we're going to miss these things :-).

Again, because this one will be so short lived it really doesn't matter in this instance.